web/src/api.ts

```diff
@@ -71,6 +71,6 @@ export async function rawQuery(query: string) {
 
 export async function getGraph(pkgName: string) {
   return await rawQuery(
-    `g.V().filter{it.get().value('pname').matches('${pkgName}')}.repeat(outE().otherV().simplePath()).times(2).path().by('pname').by(label)`
+    `g.V().has('package','pname','${pkgName}').repeat(outE().otherV().simplePath()).until(outE().count().is(0).or().loops().is(gte(2))).path().by('pname').by('label').limit(20)`
```

Comment:
```suggestion
    `g.V().has('package','pname','${pkgName}').repeat(outE().otherV().simplePath()).until(outE().count().is(0).or().loops().is(gte(2))).path().by('pname').by('label')`
```
Limiting the output here without telling the user would lead the user to erroneous analysis